tests/qam-kgraft/update_kgraft.pm

```diff
@@ -81,6 +81,8 @@ sub run() {
     # check if automounter works
     check_automounter;
 
+    script_run("while pgrep packagekitd; do pkcon quit; sleep 1; done");
```

Comment:
not a cool idea. pkcon quit is actually calling packagekitd through dbus - which auto activates packagekitd. So you might have seen the packagekitd's last moment of quitting and then autoactivate it . Saver is actually to call pkcon quit and then while pgrep; sleep; done